src/sentry/static/sentry/app/views/groupDetails/shared/header.jsx

```diff
@@ -217,24 +217,42 @@ const GroupHeader = createReactClass({
           >
             {t('Details')}
           </ListLink>
-          <ListLink to={`${baseUrl}${groupId}/activity/${this.context.location.search}`}>
+          <ListLink
+            to={`${baseUrl}${groupId}/activity/${this.context.location.search}`}
```

Comment:
I know it's old code but can we replace `this.context` as well so that `location` is used the same way?

Reply:
Sure